CONTRIBUTING.md

```diff
@@ -80,8 +80,8 @@ Whenever you want to contribute to any project. It is best practice that you sho
 
 ## 📌 Setting up project using `npm` :
 
-1. Open this clonned folder in text editor of your choice.
-2. If you want to use the project using `npm` then that comes along side when you download and install node js.
+1. Open this clonned folder in text editor of your choice
+2. If you want to use the project using `npm` then that comes along side when you download and install node js (LTS version prefered)
```

Comment:
I think it is understood, and this will just make the doc more verbose.
What I think is we can skip this change